grammar/Matlab.bnf

```diff
@@ -110,6 +111,8 @@ meta block ::= (<<p>> NEWLINE *)*
 private block_that_recovers_until_end ::= <<block element>> { recoverWhile=not_end_or_oef }
 private not_end_or_oef ::= !( NEWLINE* ( end | <<eof>> ) )
 
+global_block ::= global br* IDENTIFIER (br* IDENTIFIER)* [',' | sep] { pin=1 }
```

Comment:
Let's rename the rule to `global_variable_declaration`
Also you do not need `[',' | sep]` at the end because this suffix is in `element` rule

Reply:
I think `global_variable_declarations` would be better ('s' on the end)